at.vintagestory.VintageStory.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Should be contributed upstream if possible, along with the metainfo file & icon

Reply:
Upstream already has a similar desktop file and icon. The issue is that they are in the extra-data file, so are not available during build time.

That, and their icon is in xpm format which I think is not supported.